source/mqtt_client_connection.c

```diff
@@ -441,6 +441,49 @@ napi_value aws_napi_mqtt_client_connection_new(napi_env env, napi_callback_info
         }
     }
 
+    /* set reconnect min/max times. beware that user that might have passed only one of these values */
+    napi_value node_reconnect_min_sec = *arg++;
+    napi_value node_reconnect_max_sec = *arg++;
+    if (!aws_napi_is_null_or_undefined(env, node_reconnect_min_sec) &&
```

Comment:
Isn't it impossible for these to be null/undefined now?

Reply:
No as the MQTTConnection should set the value to default when call the function. The binding function will also threw an error message in case there are any changes on node side that removed the max/min value.